test/powershell/engine/Module/TestModuleManifest.ps1

```diff
@@ -0,0 +1,13 @@
+Describe "Test-ModuleManifest tests" -tags "CI" {
+
+    It "module manifest containing paths with backslashes or forwardslashes are resolved correctly" {
+
+        New-Item -ItemType Directory -Path testdrive:/module
+        New-Item -ItemType Directory -Path testdrive:/module/foo
+        New-Item -ItemType Directory -Path testdrive:/module/bar
+        "" > testdrive:/module/foo/bar.psm1
+        "" > testdrive:/module/bar/foo.psm1
+        New-ModuleManifest -NestedModules foo\bar.psm1,bar/foo.psm1 -RootModule foo\bar.psm1 -RequiredAssemblies foo/bar.psm1,bar\foo.psm1 -Path testdrive:/module/test.psd1
+        Test-ModuleManifest -Path testdrive:/module/test.psd1 -ErrorAction Stop | Should BeOfType System.Management.Automation.PSModuleInfo
```

Comment:
The semantics of test-modulemanifest seems to only verify existence of files and not if they are valid.  Rather than verify the files exist and being validated, I'm adding a negative test where test-modulemanifest should fail if any paths don't exist